api/schemas/flow-core.json

```diff
@@ -75,11 +76,13 @@
     },
     "avg_bit_rate": {
       "description": "The average bit rate of the flow content in 1000 bits/second. A precise definition of how the bit rate is calculated is not provided here.",
-      "type": "integer"
+      "type": "integer",
+      "exclusiveMinimum": 0
```

Comment:
I could foresee an average and maximum bit rate of 0 for Flows that don't yet have content or below 1kb/s.

Reply:
Would this be correct usage? Arguably this parameter shouldn't be set at all until the content becomes available? I guess some infrequent data  flows may have extremely low data rates, though.

Reply:
I don't think a bit rate value 0 would cause issues for a process reading the content. Ideally if the bit rate is unknown, because there is no content or it is something that couldn't be parsed or calculated, then it is omitted. A TAMS store could also decide to set it to 0 rather than deleting the property after deleting all segments.